docs/content-crag/guides/dagster/example_project.mdx

```diff
@@ -5,9 +5,9 @@ description: This guide describes our example Dagster project
 
 # Hacker News Example Project
 
-The [Hacker News example project](https://github.com/dagster-io/dagster/tree/master/examples/hacker_news) takes advantage of a wide range of Dagster features and can be useful as a point of reference for how we imagine structuring Dagster projects and using different APIs. At a high level, it consists of three pipelines, all centered around a (contrived) use case of downloading Hacker News user activity data, loading it into a database, and doing analysis / training an ML model on the aggregated data.
+The [Hacker News example project](https://github.com/dagster-io/dagster/tree/master/examples/hacker_news) takes advantage of a wide range of Dagster features and can be useful as a point of reference for how we imagine structuring Dagster projects and using different APIs. At a high level, it consists of three jobs, all centered around a (contrived) use case of downloading Hacker News user activity data, loading it into a database, and doing analysis / training an ML model on the aggregated data.
```

Comment:
I think this should be "graphs", not "jobs".  Each graph might have multiple jobs - one for each environment.